notebooks/correlation_vs_overlap.ipynb

```diff
@@ -0,0 +1,1735 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.        merged = group1.merge(group2, on='doi')
</pre><p>Interesting! I didn't know you could merge groupby objects.</p><p> </p><p>Or maybe at this point they're just dataframes (subsets of the original dataframe)? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/indices/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>